acstools/utils_calib.py

```diff
@@ -464,3 +465,81 @@ def _nint(x):
     else:
         y = x - 0.5
     return int(y)
+
+
+def is_weird_subarray(filename):
+    """
+    See if the given WFC RAW file is a custom subarray CALACS cannot process.
+    Only custom subarray with overscan might be weird.
+    If that is not the case, then perhaps OSCNTAB reference file should
+    be updated instead.
+
+    Parameters
+    ----------
+    filename : str
+        The RAW file to test.
+
+    Returns
+    -------
+    is_weird : bool
+        `True` means pipeline should not process this, but should instead
+        but manually processed and then hand-deliver to pipeline.
+
+    Examples
+    --------
+    >>> import glob
+    >>> from acstools.utils_calib import is_weird_subarray
+    >>> filenames = glob.glob('*raw.fits')
+    >>> for f in filenames:
+    ...     print(f, is_weird_subarray(f))
+    j8dm03fxq_raw.fits True
+    j8eua7qqq_raw.fits False
+
+    """
+    is_weird = False
+
+    # Dimension of fullframe dark on each chip.
+    max_y = 2048
+    max_x = max_y * 2
+
```

Comment:
If the true maximum *full* fullframe values can be obtained from the FITS header, it would be better to do this rather than hardcode dimensions.